independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/providers/serialisers/FilePartBodyHandler.java

```diff
@@ -0,0 +1,31 @@
+package org.jboss.resteasy.reactive.common.providers.serialisers;
+
+import java.io.BufferedInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Type;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.ext.MessageBodyWriter;
+import org.jboss.resteasy.reactive.FilePart;
+
+public class FilePartBodyHandler implements MessageBodyWriter<FilePart> {
```

Comment:
Wouldn't it be best to suffix these with `BodyWriter` instead of `BodyHandler` which is rather overloaded?

Reply:
I copied the other class names, but yeah, in this case they're write-only, so good point.